src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -119,6 +119,28 @@ internal static bool HasUnsafeParameter(this Symbol member)
             return false;
         }
 
+        public static bool IsEventOrPropertyWithNonPublicAccessor(this Symbol symbol)
+        {
+            switch (symbol.Kind)
+            {
+                case SymbolKind.Property:
+                    var propertySymbol = (PropertySymbol)symbol;
+
+                    bool AccessorIsNotPublic(MethodSymbol accessor)
```

Comment:
Both were intentional choices. I prefer to keep them unless they violate some guidelines.

---
In reply to: [116834659](https://github.com/dotnet/roslyn/pull/19549#discussion_r116834659) [](ancestors = 116834659)